deployment/README.md

```diff
@@ -1,379 +1,466 @@
+    `{PREFIX}-{ENV}-cicd-viewers@{DOMAIN}` | Members of this group can view CICD results in Cloud Build, for example the results of the `terraform plan` presubmit and `terraform apply` postsubmit
+    `{PREFIX}-{ENV}-bastion-accessors@{DOMAIN}` | Members of this group have permission to access the [bastion host](https://cloud.google.com/solutions/connecting-securely#bastion) project, which provides access to the private Cloud SQL instance
+    `{PREFIX}-{ENV}-project-owners@{DOMAIN}` | Members of this group have owners access to each of the deployment’s projects
+1. Add the user account that you will be using for deployment to these groups (if it is not already a member)
+
+### Set up your environment
+
+1. You can work in an existing environment, or you can configure a new environment by [creating](https://cloud.google.com/compute/docs/instances/create-start-instance) a VM instance in the Google Cloud project of your choice
+1. Confirm you have the following dependencies installed and added to your `$PATH`:
+    - [Install](https://cloud.google.com/sdk/docs/install) the Google Cloud command line tool `gcloud` (already installed if using a Google Compute Engine VM), for example:
+         ```bash
+         apt-get install google-cloud-sdk
+         ```
+    - [Install](https://cloud.google.com/storage/docs/gsutil_install) the Cloud Storage command line tool `gsutil` (already installed if using a Google Compute Engine VM)
+    - [Install](https://kubernetes.io/docs/tasks/tools/install-kubectl) the Kubernetes command line tool `kubectl`, for example:
+         ```bash
+         sudo apt-get update && sudo apt-get install -y apt-transport-https gnupg2 curl && \
+           curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add - && \
+           echo "deb https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee -a /etc/apt/sources.list.d/kubernetes.list && \
+           sudo apt-get update && \
+           sudo apt-get install -y kubectl
+         ```
+    - Install [Terraform 0.12.29](https://learn.hashicorp.com/tutorials/terraform/install-cli), for example:
+         ```shell
+         sudo apt-get install software-properties-common -y && \
+           curl -fsSL https://apt.releases.hashicorp.com/gpg | sudo apt-key add - && \
+           sudo apt-add-repository "deb [arch=amd64] https://apt.releases.hashicorp.com $(lsb_release -cs) main" && \
+           sudo apt-get update && sudo apt-get install terraform=0.12.29
+         ```
+    - Install [Go 1.14+](https://golang.org/doc/install), for example:
+         ```shell
+         sudo apt install wget -y && \
+           wget https://golang.org/dl/go1.15.6.linux-amd64.tar.gz && \
+           sudo tar -C /usr/local -xzf go1.15.6.linux-amd64.tar.gz && \
+           export PATH=$PATH:/usr/local/go/bin
+         ```
```

Comment:
why is this needed? I went through these steps on my mac and don't recall installing go.

Reply:
I think this was in the original instructions that Xin put together. If it's not necessary I will remove.